tests/beta/assistants/test_assistants.py

```diff
@@ -1,22 +1,98 @@
+from unittest.mock import patch
+
+import openai
 import pytest
 from marvin.beta.assistants import Assistant
-from marvin.utilities.openai import get_openai_client
+
+client = openai.AsyncClient()
```

Comment:
hmm do we need to stick an `api_key` kwarg in here from the settings?

Reply:
Ah yeah this must have picked up my env var locally